lib/scss/components/news-card/index.scss

```diff
@@ -1,8 +0,0 @@
-@charset 'UTF-8';
```

Comment:
@boznik @jenbreese I couldn't find where this is being used within this module. Was this just leftover copy pasta from something?

Reply:
Yes I believe so.